CHANGELOG.rst

```diff
@@ -6,6 +6,19 @@ Changelog
 
     The Dutch version of this changelog can be found :ref:`here <changelog-nl>`.
 
+3.1.4 (2025-07-11)
+==================
+
+Regular bugfix release.
+
+* Fixed broken link.
+* Updated backend dependencies:
```

Comment:
Should also add "fixed crash when saving DigiD or eHerkenning configuration in the admin" from #5394